clickhouse/migrations/1756316678212_post_analytics.up.sql

```diff
@@ -0,0 +1,81 @@
+-- up
```

Comment:
This is already battle tested right? It exists but we just add a proper migration now?